app-backend/migrations/src_migrations/main/scala/54.scala

```diff
@@ -0,0 +1,15 @@
+import slick.driver.PostgresDriver.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M54 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(54)(List(
+    sqlu"""
+    CREATE TABLE organization_features (
+      organization UUID NOT NULL UNIQUE,
+      feature_flags UUID[],
```

Comment:
I had intended to just fold the values in `organization_features.feature_flags` (if any existed) onto the full list of feature flags using a `FeatureFlag -> Option[FeatureFlagID] -> Option[FeatureFlag]` style mapping, then just bind `Some`s to `enabled` and `None`s to `disabled` in the org's config.

Should that "state" be more explicit in the DB, as opposed to being built up in the service layer?